src/redux/books/books.js

```diff
@@ -14,16 +14,16 @@ const DEFAULT_BOOKS = [
     genre: 'Action',
     id: 1,
   },
-  {
-    title: 'The Hunger Games',
-    author: 'Suzanne Collins',
-    genre: 'Action',
-    id: 2,
-  },
   {
     title: 'Dune',
     author: 'Frank Herbert',
     genre: 'Science Fiction',
+    id: 2,
+  },
+  {
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    genre: 'Economy',
     id: 3,
```

Comment:
- [ ] (OPTIONAL) Kindly set your initial state to be an empty array as per the requirements 👍